ci/launch_cluster.sh

```diff
@@ -33,24 +48,35 @@ wget 'https://downloads.dcos.io/dcos-test-utils/bin/linux/dcos-launch' && chmod
 envsubst <<EOF > config.yaml
 ---
 launch_config_version: 1
-template_url: $TEMPLATE
 deployment_name: $DEPLOYMENT_NAME
-provider: aws
+platform: aws
+instance_type: m4.large
+os_name: cent-os-7-dcos-prereqs
 aws_region: us-west-2
+provider: onprem
+installer_url: $INSTALLER_URL
 key_helper: true
-template_parameters:
-    DefaultInstanceType: m4.large
-    AdminLocation: 0.0.0.0/0
-    PublicSlaveInstanceCount: 1
-    SlaveInstanceCount: 3
-    LicenseKey: $DCOS_LICENSE
+num_masters: 3
+fault_domain_helper:
+  LocalZone:
+    num_zones: 2
+    num_public_agents: 1
+    num_private_agents: 2
+    local: true
+  RemoteZone:
+    num_zones: 2
```

Comment:
Could you help me understand. What does `num_zones: 2` mean. Two zones or two instances for this zone? How is the remote zone named?

Reply:
Ha, I think the name got you mixed-up. Yeah, I need to change the name from `RemoteZone` to `RemoteRegion` 😄 

Effectively you have two regions. A `Local` region (that becomes the default if there are no region/zone constraints) and a `Remote` region. Now for both of these regions, you define in how many zones to divide it.

My key takeway from this is : Rename `RemoteZone` to `RemoteRegion`